### PR TITLE
Syndie kit lootbox

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

A completely bullshit lootbox kit for traitors that can contain between 5 - <100 TC worth of items. Will you get lucky, or will you get fucked over?

TC amount possibilities:
Rolls a random number between 1 and 20 for the X, the TC output is:
image

Here are some items I got from the kits:
image
image
image
image
image
image
image
image
image

Fuck I am addicted.

## Why It's Good For The Game

New absolute meme of a syndie kit.
Gunna be honest, this is one of the stupidest ideas I have ever had.

## Changelog
:cl:
add: Syndie kit lootbox
fix: mega syndie kit is restricted on nuke ops again.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Done

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -273,6 +274,26 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+	desc = "A dusty crate from the back of the Syndicate warehouse. Rumored to contain a valuable assortment of items, \
+			With their all new kit, codenamed 'scam' the syndicate attempted to extract the energy of the die of fate to \
+			make a loot-box style system but failed, so instead just fake their randomness using a corgi to sniff out the items to shove in it.\
+			Item price not guaranteed. Can contain normally unobtainable items."
+	check_include_modes = FALSE
+	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops, /datum/game_mode/incursion)
+	player_minimum = 30
+
+/datum/uplink_item/bundles_TC/surplus/random/purchase(mob/user, datum/component/uplink/U)
+	var/value_index = rand(1, 20)
```
Comment (zeskorion) [inline code/modules/uplink/uplink_items.dm]: bro just roll it randomly between 5 and 100 instead of using the shittiest die to exist

Comment (PowerfulBacon) [reply code/modules/uplink/uplink_items.dm]: Ok I'll use some curve equation though so it's weighted towards shitter items

Comment (PowerfulBacon) [reply code/modules/uplink/uplink_items.dm]: There

**Comment by LordSaladin** [comment]
oh god nullcrates but strait outta your uplink

**Comment by PowerfulBacon** [comment]
Well null crates are more like the current crate, this one is like a null crate on steroids that will scam you half the time

**Comment by TotallyNotChief** [comment]
merge when

**Comment by PowerfulBacon** [comment]
when you merge it

**Review by zeskorion — APPROVED** [review]

**Pull request merged**